setup.cfg

```diff
@@ -5,7 +5,7 @@ ensure_newline_before_comments = True
 force_grid_wrap = 0
 include_trailing_comma = True
 known_first_party = deepsparse,sparsezoo
-known_third_party = bs4,requests,packaging,setuptools,numpy,onnx,onnxruntime,flask,flask_cors,tqdm,transformers,pydantic,click,yaml
+known_third_party = bs4,requests,packaging,setuptools,numpy,onnx,onnxruntime,flask,flask_cors,tqdm,transformers,pydantic,click,yaml,twint,colorama
```

Comment:
what is this for - do we need to add twint to our top-level too?

Reply:
these are for isort for our make style, just need to be added here